src/app/components/mnoe-sub-tenants-list/mnoe-sub-tenants-list.coffee

```diff
@@ -0,0 +1,95 @@
+#
+# Mnoe organizations List
+#
+@App.component('mnoeSubTenantsList', {
+  templateUrl: 'app/components/mnoe-sub-tenants-list/mnoe-sub-tenants-list.html',
+  bindings: {
+    view: '@',
+  }
+  controller: ($filter, $log, toastr, MnoeSubTenants, MnoConfirm, MnoeObservables, OBS_KEYS) ->
+    vm = this
+
+    vm.listOfSubTenant = []
+
+    # Manage sorting, search and pagination
+    vm.callServer = (tableState) ->
+      sort = updateSort (tableState.sort)
+      fetchSubTenants(vm.subTenant.nbItems, vm.subTenant.offset, sort)
+
+    # Update sorting parameters
+    updateSort = (sortState = {}) ->
+      sort = "name"
+      if sortState.predicate
+        sort = sortState.predicate
+        if sortState.reverse
+          sort += ".desc"
+        else
+          sort += ".asc"
+
+      # Update subTenant sort
+      vm.subTenant.sort = sort
+      return sort
+
+    # Widget state
+    vm.state = vm.view
```

Comment:
I think this could be removed as it will always be set to `all`. AS well in the template and component selector in the view.